lib/CustomFields/components/CustomFieldsForm/CustomFieldsForm.css

```diff
@@ -0,0 +1,30 @@
+.custom-fields-form-content {
+  margin: 0 auto;
+  max-width: 80rem;
+}
+
+.inputWrapper {
+  display: block;
+  margin-bottom: 3em;
+}
+
+.accordionNameInput {
+  font-size: 1.8em !important;
```

Comment:
Do we really need these two importants? That and all of this hardcoding of values is setting off a bunch of warning lights for me. We've got things like `variables.css` and `Layout` precisely to avoid this kind of CSS.

Reply:
I need this `!important` because some styles in `stripes-components` use nested